<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>occQtClass</class>
 <widget class="QMainWindow" name="occQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IsoAlgo3d</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/IsoAlgoGui.ico</normaloff>Resources/IsoAlgoGui.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadPCF"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom"/>
    <addaction name="actionPan"/>
    <addaction name="actionRotate"/>
    <addaction name="separator"/>
    <addaction name="actionHome"/>
    <addaction name="actionFitAll"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="IsoAlgo3d.qrc">
     <normaloff>:/Resources/exit.png</normaloff>:/Resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit IsoAlgo3d</string>
   </property>
   <property name="statusTip">
    <string>Exit IsoAlgo3d</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="IsoAlgo3d.qrc">
     <normaloff>:/Resources/lamp.png</normaloff>:/Resources/lamp.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About IsoAlgo3d</string>
   </property>
   <property name="statusTip">
    <string>About IsoAlgo3d</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="icon">
    <iconset resource="IsoAlgo3d.qrc">
     <normaloff>:/Resources/Zoom.png</normaloff>:/Resources/Zoom.png</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="toolTip">
    <string>Zoom the 3d view</string>
   </property>
   <property name="statusTip">
    <string>Zoom the 3d view</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="icon">
    <iconset resource="IsoAlgo3d.qrc">
     <normaloff>:/Resources/Pan.png</normaloff>:/Resources/Pan.png</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
   <property name="toolTip">
    <string>Pan the 3d view</string>
   </property>
   <property name="statusTip">
    <string>Pan the 3d view</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="icon">
    <iconset resource="IsoAlgo3d.qrc">
     <normaloff>:/Resources/Rotate.png</normaloff>:/Resources/Rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="toolTip">
    <string>Rotate the 3d view</string>
   </property>
   <property name="statusTip">
    <string>Rotate the 3d view</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="IsoAlgo3d.qrc">
     <normaloff>:/Resources/Home.png</normaloff>:/Resources/Home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="toolTip">
    <string>Reset the 3d view</string>
   </property>
   <property name="statusTip">
    <string>Reset the 3d view</string>
   </property>
  </action>
  <action name="actionFitAll">
   <property name="icon">
    <iconset resource="IsoAlgo3d.qrc">
     <normaloff>:/Resources/FitAll.png</normaloff>:/Resources/FitAll.png</iconset>
   </property>
   <property name="text">
    <string>Fit All</string>
   </property>
   <property name="toolTip">
    <string>Fit All for the 3d view</string>
   </property>
   <property name="statusTip">
    <string>Fit All for the 3d view</string>
   </property>
  </action>
  <action name="actionLoadPCF">
   <property name="icon">
    <iconset resource="IsoAlgo3d.qrc">
     <normaloff>:/Resources/open.png</normaloff>:/Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Load PCF</string>
   </property>
   <property name="toolTip">
    <string>Load PCF file</string>
   </property>
   <property name="statusTip">
    <string>Load PCF file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IsoAlgo3d.qrc"/>
 </resources>
 <connections/>
</ui>
